docs/tutorials/advanced-techniques-for-qaoa.ipynb

```diff
@@ -135,8 +135,7 @@
     }
    ],
    "source": [
-    "# QiskitRuntimeService.save_account(channel=\"ibm_quantum\", token=\"<MY_IBM_QUANTUM_TOKEN>\", overwrite=True, set_as_default=True)\n",
-    "service = QiskitRuntimeService(channel=\"ibm_quantum\")\n",
+    "service = QiskitRuntimeService()\n",
```

Comment:
Note that I've removed the IBM Quantum channel part, since Cloud will soon (but not yet) be the only channel - does this cause any problem with the code?